configure.ac

```diff
@@ -118,6 +118,11 @@ DOSEMU_LDFLAGS="-Wl,-warn-common -pthread"
 
 AC_CHECK_LIB(m, pow)
 
+AC_CHECK_LIB(readline, main,,[
+    AC_MSG_ERROR([No libreadline found, please install libreadline-dev package])
```

Comment:
How difficult would it be to ifdef the readline stuff?
If too difficult or a lot of code duplication, then just
don't build dosdebug when readline is missing?
But a hard dep of entire project on readline just for this...